documentation/src/main/asciidoc/userguide/appendices/LegacyBasicTypeResolution.adoc

```diff
@@ -0,0 +1,346 @@
+:sourcedir: ../../../../test/java/org/hibernate/userguide/mapping
+:extrasdir: extras
+:originalextrasdir: ../chapters/domain/extras
+
+[[basic-legacy]]
+== Legacy resolution
+
+Versions prior to 6.0 statically combined the `JavaTypeDescriptor`, `JdbcTypeDescriptor`, `BasicValueConverter` and
+`MutabilityPlan` aspects within the `BasicType` contract.  To influence this mapping, users needed to specify the
+'BasicType' or `UserType` (which gets wrapped in a `BasicType`) to use instead
+
+That said, both `BasicType` and `UserType` continue to be fully supported.  See <<basic-legacy>> for details.
```

Comment:
I would remove `See <<basic-legacy>> for details.` , we are already in this section.

Reply:
Its even worse - it points to itself! 

LOL